dev.mariinkys.Oboete.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "dev.mariinkys.Oboete",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "oboete",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network"
```

Comment:
Why do you need network? From the description it sounds like this should not be required.